src/components/ProfileCard.tsx

```diff
@@ -0,0 +1,280 @@
+    })
+  }
+
+  const hasIssue = (str: string) => {
+    return str != 'Success' && str != 'Not Updated'
+  }
+
+  return (
+    <Card
+      sx={{
+        p: 4,
+        width: '32%',
+        position: 'relative',
+      }}
+    >
+      <IconButton
+        onClick={toggleEdit}
+        sx={{
+          position: 'absolute',
+          top: 15,
+          right: 15,
+          borderBottom: '1px solid #04364A',
+          borderRadius: 0,
+          padding: 0.5,
+          paddingLeft: 1,
+        }}
+      >
+        {isEditing ? (
+          <React.Fragment>
+            <Typography color={'#04364A'}>Done</Typography>
+            <Box width={'5px'}></Box>
+            {isWaiting ? (
+              <CircleLoadAnimation />
+            ) : (
+              <DoneIcon
+                style={{ fill: '#04364A', fontSize: '24px' }}
+              ></DoneIcon>
+            )}
+          </React.Fragment>
+        ) : (
+          <React.Fragment>
+            <Typography color={'#04364A'}>Edit</Typography>
+            <Box width={'5px'}></Box>
+            <EditIcon style={{ fill: '#04364A', fontSize: '24px' }}></EditIcon>
+          </React.Fragment>
+        )}
+      </IconButton>
+      <Container
+        sx={{
+          width: '350px',
+          display: 'flex',
+          gap: '10px',
+          flexDirection: 'column',
+          alignItems: 'center',
+          marginBottom: '20px',
+        }}
+      >
+        <Avatar src="" sx={{ width: '150px', height: '150px' }} />
+        <FlexBox>
+          <ProfileButton>Change</ProfileButton>|
+          <ProfileButton>Remove</ProfileButton>
+        </FlexBox>
+        <Typography fontSize={32} fontWeight={600}>
+          {profileData?.name}
+        </Typography>
+      </Container>
+      <Stack>
+        {/* Username */}
+        <div style={{ padding: '5px' }}>
+          <hr style={{ borderTop: '1px' }} />
+          <Grid container columnSpacing={2} alignItems="center" pl={2} pr={2}>
+            <Grid item xs={4}>
+              <div style={{ width: '100%', textAlign: 'right' }}>
+                <Typography fontSize={20}>Username:</Typography>
+              </div>
+            </Grid>
+            <Grid item xs={8}>
+              {isEditing ? (
+                <TextField
+                  value={
+                    hasIssue(usernameStatus)
+                      ? oldProfileData.username
+                      : profileData?.username
+                  }
+                  onChange={(e) => {
+                    onTextFieldChange('username', e.target.value)
+                  }}
+                  inputProps={{
+                    sx: { padding: '2px 5px', fontSize: '20px' },
+                  }}
+                  disabled={isWaiting}
+                ></TextField>
+              ) : (
+                <Typography fontSize={20}>{profileData?.username}</Typography>
+              )}
+            </Grid>
+            {hasIssue(usernameStatus) ? (
```

Comment:
I think you can do `{hasIssue(usernameStatus) && <Component />}` to avoid the empty React Fragment.